analysis/extreme_events_study/visualization/README.md

```diff
@@ -0,0 +1,41 @@
+# Visualize Extreme Events
+
+## Event Display
+
+Copy the `.ig` file to local. Open the file in [iSpy WebGL](https://ispy-webgl.web.cern.ch/).
+
+## File Information
+
+* njets: 12 jets event
+* nleps: 4 leptons event with top lepton pt (553GeV)
+* pt_j: Top 2 jet pt (1.8TeV, 1.6TeV), top S_T (3.7TeV), and top H_T (3.5TeV) event
+
+## Procedure to get the `.ig` file
+
+* Modify `find_file.py` to generate a dataframe of selected event(s) from the skimmed file(s).
+* Modify `run_extreme_events.py` to run the find file processor (replace `extreme_events` with `find_file` in the script). 
+* Run `find_file.py` on the non-skimmed data `data_samples.cfg`.
+* Get run, luminosityBlock, event, and the non-skimmed root file of the event from the output dataframe.
+* Use `dasgoclient` to find the AOD file:
+
+```
+dasgoclient -query parent file=/store/data/Run2017B/DoubleMuon/NANOAOD/path_of_non_skimmed_root_file.root
+
+# Use one file from the output above
+dasgoclient -query parent file=/store/data/Run2017B/DoubleMuon/MINIAOD/path_of_root_file_from_above.root
+
+dasgoclient -query dataset file=/store/data/Run2017B/DoubleMuon/AOD/path_of_root_file_from_above.root
+
+dasgoclient -query file,run,lumi dataset=/output/from/above
+```
+
+The last output would be the AOD file, run, and a list of luminosityBlock. The run and one of the luminosityBlock have to match the selected event.
+* Compile iSpy following the instructions [here](https://github.com/cms-outreach/ispy-analyzers). For the 3 events in the folder, the matching CMSSW version is CMSSW_10_6_2.
```

Comment:
Were you able to follow these instructions exactly as they're written (besides the CMSSW change) or were there any other changes required? Also, for documentation purposes, I think it would be good to write which commit of the repository you were on

Reply:
Yes, I was able to follow that exactly. Would there be a better way than directly including the commit id in the sentence?

Reply:
I'm glad to hear no additional modifications were required! I think it'd still be useful to include the commit ID (so that in the future if someone is trying to follow your directions and it's not working, they can try checking out the commit you were on, and that will help to narrow in on where the problem is coming from).

Reply:
Sorry, I think I didn't ask my question clearly (but thank you for the explanation). Should I directly paste the commit id to the sentence?